Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "PDMats"
 uuid = "90014a1f-27ba-587c-ab20-58faa44d9150"
-version = "0.11.3"
+version = "0.11.4"
```

Comment:
Should this have bumped to 0.11.5?  It looks like 0.11.4 was already released when this was merged: https://github.com/JuliaStats/PDMats.jl/releases/tag/v0.11.4

Reply:
Yes, it should, JuliaRegistrator noticed it as well: https://github.com/JuliaStats/PDMats.jl/commit/2f3a83e4f27fae1ba9592f330a9cbaec4bb60284#commitcomment-60431201 The PR was created before the other PR was merged and released, so the diff in the PR was outdated and based on an earlier version of the master branch.

Reply:
I fixed it: https://github.com/JuliaStats/PDMats.jl/commit/15fecad14e700e024d359733d101c992707e981b :slightly_smiling_face: